src/styles/layout.sss

```diff
@@ -41,19 +41,17 @@
 .main
     flex: 5 0 60%
     display: flex
-    box-sizing: border-box
+    flex-direction: row
+    overflow: scroll
```

Comment:
Scroll should not be visible always! And definitely we ain't supposed to have horizontal scroll.

Reply:
Also should be `auto` not `scroll`

Reply:
The problem here is that I don't see scroll on mac.